chapter_introduction/index_vn.md

```diff
@@ -1457,7 +1466,12 @@ Many of these problems about how to deal with censoring,
 incentives, and feedback loops, are important open research questions.
 -->
 
-*dịch đoạn phía trên*
+Mặc dù có giá trị kinh tế lớn, các hệ thống gợi ý được xây dựng đơn thuần theo các mô hình dự đoán cũng có những hạn chế nghiêm trọng.
+Ban đầu, ta chỉ quan sát các *phản hồi được kiểm duyệt*.
+Người dùng thường có xu hướng đánh giá các bộ phim họ thực sự thích hoặc ghét: bạn có thể để ý rằng các bộ phim nhận được rất nhiều đánh giá 5 và 1 sao nhưng rất ít các bộ phim với 3 sao.
+Hơn nữa, thói quen mua hàng hiện tại thường là kết quả của thuật toán gợi ý tại thời điểm đó, nhưng các thuật toán gợi ý không luôn khai thác được chi tiết này.
+Bởi vậy có khả năng xảy ra các vòng phản hồi luẩn quẩn ở đó một hệ thống gợi ý đẩy lên một sản phẩm rồi sau đó sản phẩm đó được trả về thường xuyên hơn bởi nó được mua nhiều hơn.
+Rất nhiều trong số các bài toán về cách xử lý với kiểm duyệt, ưu đãi và vòng phản hồi là các câu hỏi quan trọng cho nghiên cứu.
```

Comment:
incentives dịch là "ưu đãi" em thấy ko quen lắm, nhưng hiện tại ko nghĩ ra được từ nào khác; put comment ở đây để note lại.

Reply:
khuyến mại?

Reply:
"incentive" dịch là "động cơ" thì sao nhỉ?
Incentive ở đây mình đang hiểu là vấn đề "incentive" của ngừoi dùng khi đánh giá 1 sao với 5 sao, nhưng lại không có "incentive" để đánh giá các sản phẩm 3 sao.

Reply:
```suggestion
Rất nhiều trong số các bài toán về cách xử lý với kiểm duyệt, khuyến khích và vòng phản hồi là các câu hỏi quan trọng cho nghiên cứu.
```

Có khi lấy bản tiếng Trung rồi dịch ra dễ hiểu hơn.

Reply:
Vậy thì "incentives" ở đây có vẻ là động cơ. Mình hiểu theo ý của @ngcthuong thấy khá hợp lý. Một vấn đề khó khăn cần giải quyết trong các bài toán này là việc người dùng thường đánh giá khi họ thực sự thích hoặc thực sự ghét, những việc này đều do một động cơ nào đó gây ra. Mình sẽ đổi lại là "động cơ".